content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js

```diff
@@ -46,6 +46,7 @@ const CONSTANTS = {
 	CSM_HELM_V1102: "1.3.2",
```

Comment:
We should remove this if we are removing the support for `CSM_HELM_V1102`.

Reply:
Removed it. Thanks!